apps/i18n/javalab/en_us.json

```diff
@@ -21,6 +21,7 @@
   "javaExtensionMissing": "Invalid file name. File names must end in '.java'.",
   "newFile": "New File",
   "noMainMethod": "Error: your program does not contain a main method.",
+  "readonlyWorkspaceHeader": "View only",
```

Comment:
I copied this from common/en_us, is that bad practice? I felt like importing all of the strings in common/en_us seems like overkill to get this one string, is there a way to not do that?

Reply:
I'm not sure, but it seems reasonable we will include other common strings in the future, so if other labs are importing them it should be ok to do it here too.

Reply:
+1 to Molly's comment. We should generally see if the string exists in common first and use that where possible. If we do it that way, the string will already be translated, and it'll be less work for our translators. The only time I'd probably make a new key is if the string already exists in common the key is very different than our use case. i.e. If the key was something like `"studentGradingCommentPlaceholder": "View only"` we'd probably want to make our own.